web/src/markdown.ts

```diff
@@ -468,16 +568,18 @@ function handleLinkifier({pattern, matches, get_linkifier_map}) {
     return url_template.expand(template_context);
 }
 
-function handleTimestamp(time) {
+function handleTimestamp(time_string: string): string {
     let timeobject;
-    if (Number.isNaN(Number(time))) {
+    const time = Number(time_string);
```

Comment:
This function was confusing because it seemed like `time` could be either a number or a string, but on further investigation it seems like it should always be a string, coming from:

```js
    if (cap = this.rules.timestamp.exec(src)) {
      src = src.substring(cap[0].length);
      out += this.timestamp(cap[1]);
```

so I modified this function given that assumption

Reply:
I think this makes sense.